### PR TITLE
Deployment: Gauge Working Balance Helper

### PR DESCRIPTION
**Pull request by EndymionJkb**
# Deployment: GaugeWorkingBalanceHelper

A helper for the front end to help users decide whether they should checkpoint their gauges.

## Deployment tasks checklist:

- [X] Contract(s) deployed to all target networks / testnets
- [X] TX IDs are properly generated
- [N/A] Action IDs are generated after deployment (if applicable) 
- [X] Outputs for all target networks are generated, and linked in the task `readme`
- [X] Addresses files are up to date with the new deployment 
- [X] Contracts are verified in every network 
- [X] Deployments `CHANGELOG` is updated
- [X] There are no code changes of any kind

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -76,6 +76,7 @@
 - Deployed `MerkleOrchard` V2 to Mainnet, Goerli, Arbitrum and Polygon.
 - Deployed `SingleRecipientGaugeFactory` V2 to Mainnet and Goerli.
 - Deployed `ProtocolIdRegistry` to Mainnet, Goerli, Arbitrum, Polygon, Optimism and Gnosis.
+- Deployed `GaugeWorkingBalanceHelper` to all networks.
```
Comment (jubeira) [inline CHANGELOG.md:79]: all networks but BSC? :upside_down_face: 
I think the clarification is not super necessary in this case.

**Review by jubeira — APPROVED** [review]
Contracts are verified, CI is green, looks good!
Only one minor comment, but otherwise it's good to go.

**Review by jubeira — APPROVED** [review]
Contracts are verified, CI is green, looks good!
Only one minor comment, but otherwise it's good to go.

File: CHANGELOG.md (hunk 1 above)
Comment (EndymionJkb) [reply CHANGELOG.md:79]: It's "all networks" that deployed a `VotingEscrowDelegationProxy` :)

I think it's fine. Close enough for governance work! There are plenty of others with exceptions, plus the definition of "all" networks changes over time anyway as we add new ones.

The general policy is to call out when something is "mainnet only" or "L2 only", or applies to a specific and limited set of networks. People can always look at the output to see exactly where it went.

**Pull request merged**